XmlFormatterOsIndependent/Commands/CloseWindowCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Avalonia.Controls;
+using System;
+using System.Windows.Input;
+using XmlFormatterOsIndependent.DataSets;
+
+namespace XmlFormatterOsIndependent.Commands
+{
+    /// <summary>
+    /// This command will close the current window
+    /// </summary>
+    internal class CloseWindowCommand : ICommand
+    {
+        /// <inheritdoc/>
+        public event EventHandler CanExecuteChanged;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'CanExecuteChanged' or invoke it.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)